src/App.jsx

```diff
@@ -1,14 +1,26 @@
 import './App.css'
+import { UserProvider } from './1components/UserContext'
+import { Routes, Route, useLocation} from 'react-router-dom'
+import UserList from './1components/Lists/UserList'
+import { Header } from './1components/Header'
+import { HomePage } from './1pages/HomePage'
 
 
-function App() {
+
+const App = () => {
+  const location = useLocation()
+
+  const isHomePage = location.pathname.startsWith('/homepage')
+  
 
   return (
-    <>
-      <div>
-      </div>
-      <h1>NC NEWS</h1>
-    </>
+    <UserProvider>
+       {isHomePage && <Header />}
+      <Routes>
+        <Route path='/' element ={<UserList/>}/>
+        <Route path='/homepage' element = {<HomePage/>}/>
```

Comment:
Have you seen a website that has a url called `homepage`? Might make more sense to be `home` if you're going down this route